jenkins_jobs/build_host_os/script.sh

```diff
@@ -1,5 +1,5 @@
 # ISO 8601 date with nanoseconds precision
-TIMESTAMP=$(date --utc -Ins)
+TIMESTAMP=$(date --utc +'%Y-%m-%dT%H:%M:%S:%N+%Z')
```

Comment:
The problem is with the `,`. For some reason YUM translated it into an encoded space (i.e., `%20`). Not sure how it handles '.', though.